build-image/first-boot.bash

```diff
@@ -134,12 +134,11 @@ sleep 10  # Related to: https://github.com/openhab/openhabian/issues/441#issueco
 echo "OK"
 
 echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
-apt-get update &>/dev/null
```

Comment:
Why is this removed? 
Is essential to install new packages based on updated apt repositories.

Reply:
it's redundant and slows down the install process
apt update happens early on unattended install and/or whenever user select one of the upgrade options from the menu anyway

Reply:
Oh I thought this was the first instance of `apt-get update`. NVM.
I actually would like to remove all `apt-get update` beside:
a) Run in first-boot
b) Run when openhabian-config start
c) Run when a new repository is added.

I will fix that.